File: src/main/java/app/askresume/domain/locale/validator/LocaleValidator.java
package app.askresume.domain.locale.validator;

import app.askresume.domain.locale.constant.LocaleType;
import org.springframework.stereotype.Service;

@Service
public class LocaleValidator {

    public String validateLocaleType(String locale) {
        return (!LocaleType.isLocaleType(locale)) ? LocaleType.EN.name() : locale;
    }
}

File: src/main/java/app/askresume/domain/locale/validator/LocaleValidator.kt
package app.askresume.domain.locale.validator

import app.askresume.domain.locale.constant.LocaleType
import org.springframework.stereotype.Service

@Service
class LocaleValidator {

    fun validateLocaleType(locale: String): String {
        return if (!LocaleType.isLocaleType(locale))
            LocaleType.EN.name
        else
            locale
    }
}

